openslides/utils/autoupdate.py

```diff
@@ -97,9 +97,15 @@ def send_object(cls, object_url):
         object_url.
         """
         # Join network location with object URL.
+        if RUNNING_HOST == '0.0.0.0':
+            # Windows can not connect to 0.0.0.0. So connect to localhost instead
+            local_host = 'localhost'
```

Comment:
Please rename this variable to `running_host` or something like this because it is not always 'localhost' or '127.0.0.1'.